cli/azd/pkg/tools/azcli/webapp.go

```diff
@@ -75,23 +76,117 @@ func (cli *azCli) appServiceRepositoryHost(
 	return hostName, nil
 }
 
+func checkWebAppDeploymentStatus(
+	res armappservice.WebAppsClientGetProductionSiteDeploymentStatusResponse,
+) (string, error) {
+	properties := res.CsmDeploymentStatus.Properties
+	inProgressNumber := int(*properties.NumberOfInstancesInProgress)
+	successNumber := int(*properties.NumberOfInstancesSuccessful)
+	failNumber := int(*properties.NumberOfInstancesFailed)
+	totalNumber := inProgressNumber + successNumber + failNumber
```

Comment:
Move thee to line 94?  Seems like the are only used when `DeploymentBuildStatusRuntimeFailed`